src/lib/constants.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const MAGNITUDE = 4;
```

Comment:
We should change it to 18 when testnet will have enough tokens.

Reply:
Hey @davidyuk  in ApproveTransaction.vue what happens if transaction is null? I don't see you checking for this. Is this a possibilit? is vue forgiving for scenarios like this? Other code looks splendid to me

Reply:
Great, looks good